### ISSUE TITLE
Reame/docs omission - how k6 doesn't work like a browser, has no DOM, etc.

### ISSUE THREAD

**Issue by na--** [loadimpact/k6#791]
When I saw this [stackoverflow question](https://stackoverflow.com/questions/52599644/wait-on-body-element) (though users have also asked similar things in slack before), I realized that I didn't mention those details in the [README improvements](https://github.com/loadimpact/k6/pull/765) a few weeks ago and I couldn't find where we explicitly explain these things in the docs.

**Comment by ppcano** [comment]
Does the `k6-does-not` section fix this issue?

https://docs.k6.io/docs#section-k6-does-not

**Comment by na--** [comment]
Yeah, that is what I meant, only we probably need it in the `README` as well. And considering that I apparently didn't find this section in the docs last year, when I created the issue, it maybe needs to be more visible or easily discoverable in some way.
